GPU/GLES/DrawEngineGLES.cpp

```diff
@@ -191,21 +188,6 @@ void DrawEngineGLES::DestroyDeviceObjects() {
 	}
 }
 
-void DrawEngineGLES::GLLost() {
-	ILOG("TransformDrawEngine::GLLost()");
-	// The objects have already been deleted by losing the context, so we don't call DestroyDeviceObjects.
-	bufferNameCache_.clear();
-	bufferNameInfo_.clear();
-	freeSizedBuffers_.clear();
-	bufferNameCacheSize_ = 0;
```

Comment:
Hmm, where do we flush the buffer name cache now?

-[Unknown]

Reply:
Uh, looks like I forgot to put these back in DestroyDeviceObjects...

Reply:
Actually no, we do wipe it there.

Reply:
But, it didn't look like lost called DestroyDeviceObjects...

-[Unknown]

Reply:
Oh. It should, oops.